multi-node/aws/pkg/config/config.go

```diff
@@ -491,6 +492,9 @@ func cidrOverlap(a, b *net.IPNet) bool {
 }
 
 func withTrailingDot(s string) string {
```

Comment:
nit: I think calling this `ensureTrailingDot` would be more descriptive. I felt the need to check for myself that the trailing dot was only added if not there. Not a blocker, maybe separate PR or something.